sdk/src/beta9/cli/machine.py

```diff
@@ -177,7 +177,8 @@ def create_machine(service: ServiceClient, pool: str):
               --tailscale-url "{res.machine.tailscale_url}" \\
               --tailscale-auth "{res.machine.tailscale_auth}" \\
               --pool-name "{res.machine.pool_name}" \\
-              --provider-name "{res.machine.provider_name}"
+              --provider-name "{res.machine.provider_name}" \\
+              --gateway-url "https://{service._config.gateway_host}"
```

Comment:
This would need to conditionally added since in `beam` the grpc host and the http host are on different subdomains